mittab/apps/tab/migrations/0011_auto_20191013_1255.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.1.5 on 2019-10-13 12:55
```

Comment:
This should be removed from this PR, right?

Reply:
Done